raft.go

```diff
@@ -937,8 +1110,8 @@ func (r *Raft) leaderLoop() {
 				v.respond(nil)
 			}
 
-		case p := <-r.peerCh:
-			p.respond(ErrLeader)
+		case future := <-r.configurationChangeChIfStable():
```

Comment:
To answer that, here are the variables in the condition for configurationChangeChIfStable(). The question becomes: can these change outside of leaderLoop() while this server is still leader?
 - r.configurations.latestIndex
 - r.configurations.committedIndex
 - r.getCommitIndex()
 - r.leaderState.commitment.startIndex (const)

I don't think so, right?